server/test/init.test.js

```diff
@@ -0,0 +1,120 @@
+      const tempStub = sinon.stub(mongodb, 'connectDB')
+      tempStub.onCall(0).returns({ collection: () => ({ findOne: () => mockObject }), close: () => {} })
+      tempStub.onCall(1).returns({ collection: () => ({ findOne: () => mockCustomer }), close: () => {} })
+      tempStub.returns({ collection: () => ({ count: () => 1 }), close: () => {} })
+      sinon.stub(addFunction, 'addFeatureToExistingUser').returns(mockObject)
+      try {
+        const result = await initUtil.handleCustomerOnload('test', 'test', 'test', 'test')
+        expect(result).to.be.equal(mockObject)
+      } catch (e) {
+        throw e
+      } finally {
+        mongodb.connectDB.restore()
+        addFunction.addFeatureToExistingUser.restore()
+      }
+    })
+    it('Should return user if input new user and valid customer', async () => {
+      const tempStub = sinon.stub(mongodb, 'connectDB')
+      let mockCustomer = {'customerCode': 'IC9-55938-5', 'hostname': 'localhost', 'features': [ {'name': 'card-1', 'versions': [ {'version': 'A', 'percent': 0}, {'version': 'B', 'percent': 0} ]}, {'name': 'card-2', 'versions': [ {'version': 'A', 'percent': 0}, {'version': 'B', 'percent': 0} ]} ]}
+      let mockNewUser = {'ops': [ 'test' ]}
+      tempStub.onCall(0).returns({ collection: () => ({ findOne: () => null }), close: () => {} })
+      tempStub.onCall(1).returns({ collection: () => ({ findOne: () => mockCustomer }), close: () => {} })
+      tempStub.returns({ collection: () => ({ insert: () => mockNewUser, count: () => 1 }), close: () => {} })
+      try {
+        const result = await initUtil.handleCustomerOnload('test', 'test', 'test', 'test')
+        expect(result).to.be.equal('test')
+      } catch (e) {
+        throw e
+      } finally {
+        mongodb.connectDB.restore()
+      }
+    })
+  })
+
+  describe('sortFeatureByCount()', () => {
+    it('Should return error if feature is in wrong format', async () => {
+      try {
+        let featureList = []
+        await initUtil.sortFeatureByCount(featureList)
+      } catch (e) {
+        expect(e).to.be.instanceOf(InvalidArgumentError)
+      }
+    })
+    it('Should return sorted feature count is input valid feature', async () => {
+      let featureList = [ {'name': 'card-1', 'versions': [ {'version': 'A', 'percent': 0, 'count': 5}, {'version': 'B', 'percent': 0, 'count': 4} ]}, {'name': 'card-2', 'versions': [ {'version': 'A', 'percent': 0, 'count': 0}, {'version': 'B', 'percent': 0, 'count': 1} ]} ]
+      await initUtil.sortFeatureByCount(featureList)
+      let result = false
+      if (featureList[0].versions[0].version === 'B' && featureList[1].versions[0].version === 'A') result = true
+      expect(result).to.be.true
```

Comment:
const result = (featureList[0].versions[0].version === 'B' && featureList[1].versions[0].version === 'A')
expect(result).to.be.true